Clinical-Genomics/cgbeacon2
# Launch the Docker demo app on port 6000

### This PR adds | fixes:
- fix #151 

### How to test:
- Run the demo app after installing it and make sure it runs on port 6000
- Launch the docker-compose file and make sure it runs the demo app on port 6000

### Expected outcome:
- [ ] The web app should run on port 6000 on both cases

### Review:

This [version](https://semver.org/) is a:
